script.js

```diff
@@ -1 +1,382 @@
+            winsList.innerHTML = `Wins: ${playerWins}`;
+            output('Congratulations, you win!');
+            gameInfo.classList.add('outcome');
+          } else {
+            output('Click on any card to continue pairing all cards in the board.'); }
+          canClick = true;
+        }, 3000);
+
+        output('Nice! You\'ve got a match!');
+        console.log('match');
+
+        // turn this card over
+        cardElement.innerHTML = `${clickedCard.displayName}<br>${clickedCard.suitSymbol}`;
+        cardElement.classList.add('squareFace', firstCard.suitColour);
+      } else {
+        output('Oops, that wasn\'t a match.');
+        console.log('NOT a match');
+        canClick = false;
+        // turn this card over
+        cardElement.innerHTML = `${clickedCard.displayName}<br>${clickedCard.suitSymbol}`;
+        cardElement.classList.add('squareFace', firstCard.suitColour);
+
+        setTimeout(() => {
+          // turn this card back over
+          firstCardElement.innerText = '';
+          firstCardElement.classList.remove('squareFace');
+          cardElement.innerText = '';
+          cardElement.classList.remove('squareFace');
+          output('Click on any card to continue.');
+          canClick = true; }, 3000); }
+      // reset the first card
+      firstCard = null; }
+  }
+};
+
+// create all the board elements that will go on the screen
+// return the built board
+const buildBoardElements = () => {
+  // create the element that everything will go inside of
+
+  // give it a class for CSS purposes
+  boardElement.classList.add('board');
+
+  // use the board data structure we passed in to create the correct size board
+  for (let i = 0; i < board.length; i += 1) {
+    // make a var for just this row of cards
+    const row = board[i];
+
+    // make an element for this row of cards
+    const rowElement = document.createElement('div');
+    rowElement.classList.add('row');
+
+    // make all the squares for this row
+    for (let j = 0; j < row.length; j += 1) {
+      // create the square element
+      const square = document.createElement('div');
+
+      // set a class for CSS purposes
+      square.classList.add('square');
+
+      // set the click event
+      // eslint-disable-next-line
+      square.addEventListener('click', (event) => {
+        // we will want to pass in the card element so
+        // that we can change how it looks on screen, i.e.,
+        // "turn the card over"
+        squareClick(event.currentTarget, i, j);
+      });
+
+      rowElement.appendChild(square);
+    }
+    boardElement.appendChild(rowElement);
+  }
+
+  return boardElement;
+};
+
+const gameReset = () => {
+  if (canClick) {
+    clearInterval(ref);
+    timer.remove();
+    inputField.value = '';
+    boardElement.innerHTML = '';
+    numPairs = 0;
+    board.splice(0, board.length);
+    reset.remove();
+    winsList.remove();
+    // inputContainer.remove();
+    initGame();
+    inputContainer.remove();
```

Comment:
Could use a comment here to mention that you are removing unnecessary elements that are created in initGame